include/opt-sched/Scheduler/machine_model.h

```diff
@@ -163,7 +163,7 @@ class MachineModel {
   // Add a new instruction type.
   void AddInstType(InstTypeInfo &instTypeInfo);
   // Add a new issue type.
-  void AddIssueType(IssueTypeInfo &issueTypeInfo);
+  void addIssueType(IssueTypeInfo &issueTypeInfo);
```

Comment:
Nitpick: This should be `void addIssueType(IssueTypeInfo &IssueTypeInfo);`

Reply:
I implemented all the changes you suggested.